src/commonNative/kotlin/com/epam/drill/core/callbacks/classloading/ClassLoadHook.kt

```diff
@@ -84,13 +84,18 @@ fun classLoadEvent(
                     transformers += { bytes -> SSLTransformer.transform(kClassName, bytes, loader) }
                 }
             }
+        }
+
+        if (config.isKafka) {
+            val classReader = ClassReader(classBytes)
```

Comment:
this val duplicates above...
maybe it would be better to extract this condition to a new method?